### DISCUSSION TITLE
Combine using PHP-processor and HTTP-Processor issue

### DISCUSSION THREAD

**Discussion by sam8881998** [apioo/fusio#576] [Q&A]
Hi, I am looking for sql the value from the database table, say table A (select X from table A where Y=10), and then call external API using HTTP-processor with the X value. Getting the Json file and then insert the content value to table B. I was trying to use PHP-sandbox, and HTTP-Processor but cannto be figure out. Not sure the approach I am using is incorrect.  Please advise how to make it and thanks.

**Comment by chriskapp** [comment]
Hi @sam8881998 I' am not sure whether I understand your use case completely but your could solve this by using i.e. the Worker PHP Local action, this takes only a few lines s.

```php
$connection = $connector->getConnection('App'); // your SQL connection
$httpClient = $connector->getConnection('my_endpoint'); // your HTTP connection

$value = $connection->fetchOne('select X from table A where Y=10');

$httpClient->request('POST', '/my_endpoint', ['json' => ['value' => $value]);
```

**Reply by sam8881998** [reply]
Thanks for your advise.
Let me elaborate further.

1. I read the SQL table (table A) to get the $value
2. with the $value, I need to get the Json file from customer end_point http://.... /customer\:$value
3. from the Json file I get from customer end_point in point 2, and then decode json and store the values back to SQL table B.

Thanks.
